rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -714,6 +726,62 @@ int __pal_init_v1(pal_attr_v1_t *attr)
 
 	parse_args(attr->args);
 
+	if (ra_tls_server) {
+		printf("Begin server test!\n");
+
+		sgx_launch_token_t t;
+		sgx_enclave_id_t eid;
+		memset(t, 0, sizeof(t));
+
+		int updated = 0;
+		const char *enclave_path = "/run/rune/Wolfssl_Enclave.signed.so";
```

Comment:
Because ra-tls directory will be moved into skeleton so the file names are controlled by ourselves. I suggest this file name should be called "enclave-tls-wolfssl.signed.so".